robot/src/main/java/frc/robot/util/Constants/WristConstants.java

```diff
@@ -16,6 +16,8 @@ public class WristConstants {
 
     public static final double WRIST_DEGREES_PER_INCREMENT = 2.5;
 
+    public static final double WRIST_DEGREES_SUBWOOFER_POSITION = -25;
```

Comment:
I do recall using the value -25 when we were testing last Wednesday, but I'd like to start testing with a less aggressive target value and increase it as needed. We don't want to burn out the wrist motor, or stress the hardware, due to pushing the mechanism too hard against the bumper or ground. With 0 being the pickup position, and having about 10 degrees of backlash, let's start at `-15` instead.